.github/workflows/ci.yaml

```
@@ -118,4 +118,6 @@ jobs:
             otp: 27
           - elixir: 1.18
             otp: 27
+          - elixir: 1.18.4-otp-27
+            otp: 28
```

This seems off, are we sure about this pairing? The elixir version specifies OTP 27 but the OTP version is then set to 28?

Should this be updated to match the other versions?
```suggestion
          - elixir: 1.18
            otp: 28
```